packages/numeric-input/src/behaviors/SpinnerBehavior.js

```diff
@@ -0,0 +1,78 @@
+import { Component } from "react";
+import PropTypes from "prop-types";
+
+export default class SpinnerBehavior extends Component {
+  static propTypes = {
+    onClick: PropTypes.func,
+    children: PropTypes.func,
+    onChange: PropTypes.func,
+    value: PropTypes.number,
+    initialValue: PropTypes.number,
+    step: PropTypes.number,
+    disabled: PropTypes.bool
+  };
+  static defaultProps = {
+    step: 1,
+    initialValue: 0,
+    value: undefined,
+    onChange: undefined
```

Comment:
`onChange` should not be `undefined` you can default it to an empty function `() => {}`.